opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -58,39 +61,100 @@ def __init__(
         self._meter_provider = meter_provider
 
     def create_counter(self, name, unit=None, description=None) -> APICounter:
-        return Counter(self._instrumentation_info, name, unit, description)
+        return Counter(
+            self._instrumentation_info,
+            name,
+            # pylint: disable=protected-access
+            self._meter_provider._measurement_consumer,
```

Comment:
I don't think `Meter` ever actually uses `self._meter_provider` for anything but accessing the consumer. Can we instead just store a reference to the measurement_consumer, remove reference to the meter provider, and then we can fix all these pylint warnings

Reply:
I think `Meter` needs access to `MeterProvider` to be able to get the configuration changes that happen on the provider level. https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#get-a-meter

Reply:
It's not used currently, can we add it back at that point? I mostly just don't like sprinkling `# pylint: disable=protected-access` everywhere

Reply:
We could add it back later but I think the pylint lines would still be present? Either way it's fine with me to add it later. We MIGHT  not even need it in `Meter` if we discover that what is configurable is not actually mutable (this is why `Tracer` does not have a reference to `TracerProvider`, because resource, id_generator, instrumentation info and span limits don't change during runtime. As well, the span processors take in a sequence so these can be picked up during runtime.

Reply:
Right, fixed